docker/Dockerfile.multi

```
@@ -73,6 +73,10 @@ RUN TORCH_INSTALL_TYPE=${TORCH_INSTALL_TYPE} bash ./install.sh --pytorch && rm i
 
 RUN bash ./install.sh --opencv && rm install.sh
 
+# Copy and install dependencies from constraints.txt
+COPY constraints.txt /tmp/constraints.txt
+RUN pip3 install --no-cache-dir --upgrade -r /tmp/constraints.txt && rm /tmp/constraints.txt
```

```suggestion
RUN pip3 install --no-cache-dir -r /tmp/constraints.txt && rm /tmp/constraints.txt
```

But I think you don't need to rebuild the images and run the CI pipeline again.